src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -88,7 +88,7 @@ public static THeaders GetHeaders<THeaders>(this OtlpExporterOptions options, Ac
         return headers;
     }
 
-    public static OtlpExporterTransmissionHandler<TraceOtlpCollector.ExportTraceServiceRequest> GetTraceExportTransmissionHandler(this OtlpExporterOptions options)
+    public static OtlpExporterTransmissionHandler<TraceOtlpCollector.ExportTraceServiceRequest> GetTraceExportTransmissionHandler(this OtlpExporterOptions options, bool enableRetry = false)
```

Comment:
I asked above if there will be other types of retires. If yes, maybe pass in `ExperimentalOptions` here instead of `bool enableRetry` to avoid having a bunch of `bool`s needing to be passed.

Reply:
I was gonna update when I implement the other (persistent storage) retry strategy. I will change it here.